app.py

```diff
@@ -19,9 +19,14 @@ def login():
 def signup():
     return render_template('signup.html')
 
-@app.route('/lynn')
+@app.route('/lynn', methods=['GET', 'POST'])
 def lynn():
-    return render_template('lynn.html')
+    message = ""
+    if request.method == 'POST':
+        message = request.form['message']
```

Comment:
Let's add a check here that the message string shouldn't be empty because we don't want to pass empty strings to dialogFlow and ensure our system is full proof.